tests/unit/hive_metastore/test_catalog_schema.py

```diff
@@ -86,7 +94,7 @@ def prepare_test(ws, backend: MockBackend | None = None) -> CatalogSchema:
     return CatalogSchema(ws, table_mapping, principal_acl, backend)
 
 
-@pytest.mark.parametrize("location", ["s3://foo/bar", "s3://foo/bar/test"])
+@pytest.mark.parametrize("location", ["s3://foo/bar", "s3://foo/bar/test", "s3://foo/bar/test/1"])
```

Comment:
To show it is not about the integer
```suggestion
@pytest.mark.parametrize("location", ["s3://foo/bar", "s3://foo/bar/test", "s3://foo/bar/test/baz"])
```